### ISSUE TITLE
chronological order / areas of publications? 

### ISSUE THREAD

**Issue by lordgrilo** [nplresearch/nplresearch.github.io#2]
Is it possible to easily implement that the papers are listed in reverse chronological order (by adding a date/published tag), or do we need to this by hand? 

Second thing... can we define tags for example linking paper to sub-areas of research of the lab? (a little bit like it's done on the academic hugo template)?

**Comment by antonio-leitao** [comment]
Yes to both, not implemented from the beggining because of the small amount of publications.
The only discussion here would be UI related as in: how would you want to have it look like?

**Comment by lordgrilo** [comment]
I was thinking we might want to have some "areas/projects" tags, similar to what I have on my own website, given that we are starting to have quite a bit of publications. 
Re chronology, the simplest thing would be that we have a date/time record for each paper, and the website just shows things in reverse chronological order (from recent to old). 
What do you think?

**Comment by antonio-leitao** [comment]
sure. Keeping also in mind the easyness of add/removing papers i think the best way is to have separate bib files for each area. This way it's easy to add/remove and also we can have many to one relations (one paper in many areas).
Should these areas be accessible from the homepage? Or is it just a separation on the publications page (or maybe even dropping the publications page altogether and have just the projects)?
